go-controller/pkg/config/config.go

```diff
@@ -1296,6 +1304,9 @@ func initConfigWithPath(ctx *cli.Context, exec kexec.Interface, saPath string, d
 	}
 
 	// Logging setup
+	if debug == true {
+		cfg.Logging.Level = 5
```

Comment:
is it ever useful to use something != 5

if so, can we override this cfg.Logging.Level easily when running tests?

Reply:
Yep, it's really easy to override as the command line flags take precedence.
Essentially the default level is 4, debug is 5 and if you wanted something specific for a certain test you could  provide a `--loglevel=42` flag to `app.Run` and that would take precedence.